cdv/cmds/chia_inspect.py

```diff
@@ -100,15 +100,23 @@ def streamable_load(cls: Any, inputs: Iterable[Any]) -> List[Any]:
     input_objs: List[Any] = []
     for input in inputs:
         if "{" in input:  # If it's a JSON string
-            input_objs.append(cls.from_json_dict(json_and_key_strip(input)))
+            json_dict = json_and_key_strip(input)
+            parsed_obj = cls.from_json_dict(json_dict)
         elif "." in input:  # If it's a filename
             file_string = open(input, "r").read()
             if "{" in file_string:  # If it's a JSON file
-                input_objs.append(cls.from_json_dict(json_and_key_strip(file_string)))
+                json_dict = json_and_key_strip(file_string)
+                parsed_obj = cls.from_json_dict(json_dict)
             else:  # If it's bytes in a file
-                input_objs.append(cls.from_bytes(hexstr_to_bytes(file_string)))
+                original_bytes = hexstr_to_bytes(file_string)
+                parsed_obj = cls.from_bytes(original_bytes)
+                assert bytes(parsed_obj) == original_bytes  # assert the serialization incase it was only a partial read
```

Comment:
Had to add this because some objects can parse themselves from other larger serializations (ignoring the rest).  We should fail if we don't reserialize back to exactly what we were given.